src/workers/cdc.ts

```diff
@@ -193,6 +195,14 @@ const onCommentChange = async (
         data.payload.after.id,
       );
     }
+  } else if (data.payload.op === 'u') {
+    await notifyCommentEdited(logger, data.payload.after);
+  } else if (data.payload.op === 'd') {
+    await notifyCommentDeleted(logger, {
+      ...data.payload.before,
+      content: undefined,
```

Comment:
Why do you want to change these props? I think we should keep it as is. We can't be 100% sure who's gonna subscribe to this topic, and this is not an expected behavior

Reply:
I agree, but we would need a reference on which comment was deleted. The `after` is `null` once deleted, so I had to take the `before` for the IDs and applied the empty content as part of being a `null` after.

Reply:
The topic itself means this comment is deleted. It's not trivial to mess with the properties of the data. Again, someone in the future may subscribe to event and will not figure out what happens here. I suggest we find another way to indicate deletion, e.g. providing a flag to the generator

Reply:
The value of `data.payload.after` is `null`, the subscriber won't have any reference to which comment was deleted if we pass just the `after` object. We have two options:
- We will pass the `id` only in here to still have the reference on the subscription, or
- We will send a flag to the generator to clear **all** `ContentImage` with the `type` we sent with all the `referenceId` that is not present on the relevant table anymore. Might become a heavy process in the future when as we will fetch all `Comment` type (for example)  then we will query all reference IDs that do not exist on the `Comments` table anymore.

Reply:
I think you misunderstood me. My suggestion is to send `before` as you did already but without manipulating `content` and `contentHtml`. Just pass the object as is

Reply:
Oh, right. Yep. That makes sense. Let me do that 🚀